test/NinjaTools.FluentMockServer.API.Exclude.Tests/Downstream/RequestMapperSpec.cs

```diff
@@ -78,10 +75,7 @@ public RequestMapperSpec(ITestOutputHelper outputHelper) : base(outputHelper)
                 actual.Headers.Should().NotBeNull();
                 actual.Headers.ToList().Count.Should().Be(expected.Headers.ToList().Count);
 
-                if (expected.Headers.Any())
-                {
-                    actual.Headers.ToList().Should().Contain(expected.Headers.ToList());
-                }
+                if (expected.Headers.Any()) actual.Headers.ToList().Should().Contain(expected.Headers.ToList());
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/manual/git_36/NinjaTools.FluentMockServer/pullRequest?prid=4739715)